modules/vote/main.lua

```diff
@@ -149,7 +149,7 @@ local function GroupElectionResult(eventCode, electionResult, descriptor)
 					if IsUnitOnline(group_tag) then
 						CHAT_ROUTER:AddSystemMessage(strformat('|cFF0000%s %s|r', units.GetDisplayName(group_tag), GetString(HR_VOTE_NOT_READY_CHAT)))
 					else
-						CHAT_ROUTER:AddSystemMessage(zo_strformat("<<1>><<2>> (OFFLINE)", color_orange, GetUnitDisplayName(group_tag)))
+						CHAT_ROUTER:AddSystemMessage(zo_strformat("<<1>><<2>> (OFFLINE)", ZO_ORANGE, GetUnitDisplayName(group_tag)))
```

Comment:
that will crash the script. ZO_ORANGE is a table and not a string. Please remove this line here. i already fixed it here: https://github.com/m00nyONE/HodorReflexes/commit/70d4b03a7cac925c5279a1f237d354cbc5e0250d

Reply:
Sorry that was hasty, the right way to use a ZO_ColorDef is like how they use it here,

https://github.com/esoui/esoui/blob/8ff2e6779115b1342912883bbdd6db1e604b4cd5/esoui/common/globals/platformstoredialogs.lua#L142